src/de/2024-04/03/teacher-questions.md

```diff
@@ -9,6 +9,9 @@ date:   18/10/2024
 
 -   Was in diesem Abschnitt könnt ihr logisch nicht nachvollziehen?
 
+```{=html}
```

Comment:
@toak Unfortunately we can no longer accept improperly formatted files, please clean it up from using non-markdown format.